lib/simulation/openfisca/mapping/reverse.js

```diff
@@ -12,8 +12,8 @@ module.exports = function reverseMap(openFiscaResponse, situation) {
     })));
     var injectedPrestations = _.intersection(_.keys(PRESTATIONS), injectedRessources);
 
-    // Openfisca famille object, which contains the values of computed prestations
-    var openFiscaFamille = openFiscaResponse.value[0].familles[0];
+    // Computed prestations can be in the 'familles' or 'individus' openfisca entity
+    var openfiscaResults = _.merge(openFiscaResponse.value[0].familles[0], openFiscaResponse.value[0].individus[0]);
```

Comment:
Keep consistent casing of `openFisca` (I agree with it being a single word in this context and thus all lowercase, but the rest of the code considers it as two words).